src/Tizen.Nlp/Tizen.Nlp/NlpHandle.cs

```diff
@@ -0,0 +1,149 @@
+        /// </summary>
+        public event EventHandler<MessageReceivedEventArgs> MessageReceived;
+        private Message _msg;
+        private readonly Message.NotifyCb _noti = new Message.NotifyCb();
+        private string _tag;
+
+        private void MakeRequest(string cmd, string info)
+        {
+            Bundle b = new Bundle();
+            b.AddItem("command", cmd);
+            b.AddItem("info", info);
+            _msg.Send(b);
+        }
+
+        private void OnReceived(string sender, Bundle msg)
+        {
+            Log.Debug(_tag, "OnReceived ++");
+            MessageReceivedEventArgs e = new MessageReceivedEventArgs();
+            e.Message = msg;
+            MessageReceived?.Invoke(this, e);
+            Log.Debug(_tag, "done");
+        }
+
+        /// <summary>
+        /// An init session to connect remote tidl service with 2 input parameters.
+        /// </summary>
+        /// <param name="serviceId">remote nlp service id.</param>
+        /// <param name="clientId">local nlp client app id.</param>
+        /// <returns></returns>
+        /// <since_tizen> 5 </since_tizen>
+        public void Init(string serviceId, string clientId)
+        {
+            _tag = clientId;
+            Log.Debug(_tag, "msg construct started");
+            _msg = new Message(serviceId);
+            Log.Debug(_tag, "msg construct success");
+            _noti.Received += OnReceived;
+            Log.Debug(_tag, "notify callback be assigned");
+            _msg.Connected += (sender, e) =>
+            {
+                Log.Debug(_tag, "start to register");
+                _msg.CoRegister(clientId, _noti);
+                Log.Debug(_tag, "connected callback be called");
+            };
+            Log.Debug(_tag, "start to connect");
+            _msg.Connect();
+            Log.Debug(_tag, "wait to callback of onConnected");
+        }
+
+        /// <summary>
+        /// An release session to disconnect remote tidl service.
+        /// </summary>
+        /// <since_tizen> 5 </since_tizen>
+        public void Release()
+        {
+            _noti.Received -= OnReceived;
+            _msg.UnRegister();
+            _msg.Dispose();
+            _msg = null;
+
+        }
+
+        /// <summary>
+        /// Send Pos of Tag request to remote tidl service with 1 input parameters.
+        /// </summary>
+        /// <param name="info">A sentence need to be processed.</param>
+        /// <returns></returns>
+        /// <since_tizen> 5 </since_tizen>
+        public void PosTag(string info)
+        {
+            MakeRequest("pos_tag", info);
+        }
+
+        /// <summary>
+        /// Send Named Entity recognition request to remote tidl service with 1 input parameters.
+        /// </summary>
+        /// <param name="info">A sentence need to be processed.</param>
+        /// <returns></returns>
+        /// <since_tizen> 5 </since_tizen>
+        public void NeChunk(string info)
```

Comment:
X DO NOT use abbreviations or contractions as part of identifier names. 
https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/general-naming-conventions